suites/reef/rados/tier-2_rados_test-bugfixes.yaml

```diff
@@ -150,33 +150,51 @@ tests:
           plugin: jerasure
           disable_pg_autoscale: true
         inconsistent_obj_count: 4
+        replicated_pool:
```

Comment:
Why is replicated pool added here? we are covering the test for re pools separately??

Also, please add bug : https://bugzilla.redhat.com/show_bug.cgi?id=2277111 , for Inconsistent objects in EC test

Reply:
I added the bug detail in the Pacific and Quincy suites. The bug is not observed in the Reef so I have not added the bug details in the Reef suite file.